3rdparty/Makefile.am

```diff
@@ -51,7 +51,7 @@ EXTRA_DIST =		\
 EXTRA_DIST +=		\
   $(LEVELDB).patch
 
-# We need to patch ZooKeeper in order to get 3.4.5 to compile on
+# We need to patch ZooKeeper in order to get 3.4.8 to compile on
 # OS X 10.10. See: MESOS-1797.
```

Comment:
Thank your for pointing this, I will remove this comment. And for the zookeeper patch file I think I already removed it.